audio/src/channel/linear.rs

```diff
@@ -152,6 +146,13 @@ where
     }
 }
 
+impl<T> AsRef<[T]> for LinearChannel<'_, T>
+{
+    fn as_ref(&self) -> &[T] {
```

Comment:
For trivial impls it actually can have a fairly significant impact on Rust's ability to optimize (still)! I've noted this in a different project of mine ([fixed-map](https://github.com/udoprog/fixed-map)).
```suggestion
    #[inline]
    fn as_ref(&self) -> &[T] {
```